docker make build fails

Step 3/31 : RUN conda install -y -c conda-forge         tensorflow         jupyter_contrib_nbextensions
 ---> Running in 3f82f86c2ec1
Solving environment: ...working... done


==> WARNING: A newer version of conda exists. <==
  current version: 4.4.10
  latest version: 4.4.11

Please update conda by running

    $ conda update -n base conda


certifi 2018.1.18: #######5   |  75% 
tensorflow 1.1.0: ########## | 100% 
pbr 3.1.1: ########## | 100% 
jupyter_highlight_selected_word 0.1.0: ########## | 100% 
jupyter_contrib_nbextensions 0.4.0: ########## | 100% 
protobuf 3.5.1: ########## | 100% 
jupyter_nbextensions_configurator 0.4.0: ########## | 100% 
mock 2.0.0: ########## | 100% 
jupyter_latex_envs 1.4.0: ########## | 100% 
jupyter_contrib_core 0.3.3: ########## | 100% 

IsADirectoryError(21, 'Is a directory')



## Package Plan ##

  environment location: /opt/conda

  added / updated specs: 
    - jupyter_contrib_nbextensions
    - tensorflow


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    certifi-2018.1.18          |           py36_0         143 KB  conda-forge
    tensorflow-1.1.0           |           py36_0        25.3 MB  conda-forge
    pbr-3.1.1                  |           py36_0         112 KB  conda-forge
    jupyter_highlight_selected_word-0.1.0|           py36_0          13 KB  conda-forge
    jupyter_contrib_nbextensions-0.4.0|           py36_0        19.4 MB  conda-forge
    protobuf-3.5.1             |           py36_3         5.9 MB  conda-forge
    jupyter_nbextensions_configurator-0.4.0|           py36_0         470 KB  conda-forge
    mock-2.0.0                 |           py36_0         103 KB  conda-forge
    jupyter_latex_envs-1.4.0   |           py36_1         738 KB  conda-forge
    jupyter_contrib_core-0.3.3 |           py36_1          28 KB  conda-forge
    ------------------------------------------------------------
                                           Total:        52.2 MB

The following NEW packages will be INSTALLED:

    jupyter_contrib_core:              0.3.3-py36_1     conda-forge
    jupyter_contrib_nbextensions:      0.4.0-py36_0     conda-forge
    jupyter_highlight_selected_word:   0.1.0-py36_0     conda-forge
    jupyter_latex_envs:                1.4.0-py36_1     conda-forge
    jupyter_nbextensions_configurator: 0.4.0-py36_0     conda-forge
    mock:                              2.0.0-py36_0     conda-forge
    pbr:                               3.1.1-py36_0     conda-forge
    protobuf:                          3.5.1-py36_3     conda-forge
    tensorflow:                        1.1.0-py36_0     conda-forge

The following packages will be UPDATED:

    certifi:                           2018.1.18-py36_0             --> 2018.1.18-py36_0 conda-forge


Downloading and Extracting Packages
ERROR: Service 'handson-ml' failed to build: The command '/bin/sh -c conda install -y -c conda-forge         tensorflow         jupyter_contrib_nbextensions' returned a non-zero code: 1
make: *** [Makefile:9: build] Error 1

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -8,6 +8,7 @@ RUN apt-get update && apt-get upgrade -y \
         sudo \
     && rm -rf /var/lib/apt/lists/*
 
+RUN conda update -n base conda
 RUN conda install -y -c conda-forge \
         tensorflow \
         jupyter_contrib_nbextensions

